### PR TITLE
THORN-1772 : Update SmallRye FT to 1.1

### PR DESCRIPTION
**Pull request by antoinesd**
- [x] Have you followed the guidelines in our [Contributing](http://wildfly-swarm.io/community/contributing/) document?
- [x] [v2] Have you created a [JIRA](https://issues.jboss.org/browse/THORN) and used it in the commit message?
- [ ] [v4] Have you created a [GitHub Issue](https://github.com/thorntail/thorntail/issues) and used it in the commit message?
- [ ] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/thorntail/thorntail/pulls) for the same issue?
- [x] Have you built the project locally prior to submission with `mvn clean install`?

-----

### CODE REVIEW

File: fractions/microprofile/microprofile-fault-tolerance/src/main/resources/modules/io/smallrye/faulttolerance/main/module.xml [hunk 1]
```diff
@@ -25,10 +25,14 @@
...
       <module name="org.eclipse.microprofile.config.api" export="true"/>
+      <module name="io.smallrye.metrics" export="true"/>
       <module name="org.wildfly.extension.microprofile.config" export="true"/>
       <module name="org.wildfly.swarm.configuration.microprofile.config" slot="main"/>
+      <module name="org.wildfly.swarm.microprofile.metrics" slot="main"/>
       <module name="org.wildfly.swarm.microprofile.faulttolerance" slot="deployment"/>
       <module name="org.eclipse.microprofile.faulttolerance" export="true"/>
       <module name="com.netflix.hystrix" slot="main"/>
+      <module name="org.jboss.weld.core" export="true"/>
+     <module name="org.jboss.weld.spi" export="true"/>
```
Comment (Ladicek) [inline fractions/microprofile/microprofile-fault-tolerance/src/main/resources/modules/io/smallrye/faulttolerance/main/module.xml]: Indentation.

File: testsuite/microprofile-tcks/pom.xml [hunk 2]
```diff
@@ -45,6 +45,7 @@
         <version.maven-surefire>2.20.1</version.maven-surefire>
         <!-- commons-logging (needed by httpclient) is for some reason exluded -->
         <version.commons-logging>1.2</version.commons-logging>
+      <version.hamcrest>1.3</version.hamcrest>
```
Comment (Ladicek) [inline testsuite/microprofile-tcks/pom.xml]: Indentation.

**Comment by juangon** [comment]
Test failure with java.lang.NoClassDefFoundError: org/hamcrest/Matchers

**Review by mkouba — APPROVED** [review]

**Comment by antoinesd** [comment]
Thank you @mkouba. Apparently I forgot to push my last commit with Hamcrest inclusion :-(

**Comment by mkouba** [comment]
@antoinesd no problem and you're welcome ;-)

**Pull request merged**